### ISSUE TITLE
Microphone is not closed after call is ended.

### ISSUE THREAD

**Issue by leftyb** [RestComm/Restcomm-Mayday-Web#3]

**Comment by atsakiridis** [comment]
Right, this is what I get in Chrome as well. For Firefox the call hangs up but the video window is left showing 'Video format or MIME type is not supported':

![screen shot 2016-05-24 at 9 50 07 am](https://cloud.githubusercontent.com/assets/8684513/15495677/feeb7df4-2199-11e6-9565-2f3605c25524.png)
